myst_nbs/howto/howto_debugging.myst.md

```diff
@@ -6,108 +6,184 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
+(howto_debugging)=
 # How to debug a model
 
-There are various levels on which to debug a model. One of the simplest is to just print out the values that different variables are taking on.
+:::{post} July 11, 2016
+:tags: 
+:category: beginner
+:author: Thomas Wiecki
+:::
 
-Because `PyMC3` uses `Theano` expressions to build the model, and not functions, there is no way to place a `print` statement into a likelihood function. Instead, you can use the `Theano` `Print` operatator. For more information, see:  theano Print operator for this before: http://deeplearning.net/software/theano/tutorial/debug_faq.html#how-do-i-print-an-intermediate-value-in-a-function.
```

Comment:
We can keep the link to the "debug faq" page, but it should not be a hardcoded url. You can use `` {ref}`aesara:debug_faq` `` which will be rendered as a link and substituted as text containing the title of the page in the aesara docs. Or alternatively use `` {ref}`custom text <aesara:debug_faq>` `` to use custom text for the link on the website.

More info at https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_documentation/docs_presentation.html#cross-referencing-roles and there is also a webinar on the topic if interested: https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_documentation/index.html#video